plugin.xml

```diff
@@ -153,6 +153,8 @@ to config.xml in order for the application to find previously stored files.
         <source-file src="src/android/AssetFilesystem.java" target-dir="src/org/apache/cordova/file" />
         <source-file src="src/android/PendingRequests.java" target-dir="src/org/apache/cordova/file" />
 
+        <framework src="androidx.webkit:webkit:1.3.0" type="gradleReference" />
```

Comment:
The version should be configurable. And gradleReference is used for gradle files, not for maven dependencies.
You can check camera plugin for reference
https://github.com/apache/cordova-plugin-camera/blob/master/plugin.xml#L98

Reply:
https://github.com/apache/cordova-plugin-file/pull/513/commits/d11fe941db55d1d10fde7965d6fae6b3a65bfb5f